torch/_inductor/codegen/cpp.py

```diff
@@ -397,6 +407,43 @@ def expm1(x):
     def log1p(x):
         return f"{x}.log1p()"
 
+    @staticmethod
+    def masked(mask, body, other):
+        assert "is_masked_load" in V.interpreter.current_node.meta
+        assert V.interpreter.current_node.meta["is_masked_load"]
+        code = BracesBuffer()
+
+        var = V.kernel.cse.newvar()
+        if other == float("-inf"):
+            code.writeline(
+                f"auto {var} = at::vec::Vectorized<float>(-std::numeric_limits<float>::infinity());"
+            )
+        elif other == float("inf"):
+            code.writeline(
+                f"auto {var} = at::vec::Vectorized<float>(std::numeric_limits<float>::infinity());"
+            )
+        elif isinstance(other, float):
+            code.writeline(f"auto {var} = at::vec::Vectorized<float>({other});")
+        else:
+            code.writeline(f"auto {var} = at::vec::Vectorized<float>({other!r});")
```

Comment:
May I know why do we use the conversion flag "!r" here?

Reply:
"!r" means `repr` - https://docs.python.org/3/library/functions.html#repr 
I copied it from the non-vec version.